adapter/src/main/java/com/twilio/kudu/sql/TableType.java

```diff
@@ -15,11 +15,11 @@
 package com.twilio.kudu.sql;
 
 public enum TableType {
-  FACT(20000000.0d), CUBE(2000000.0), DIMENSION(1000.0), SYSTEM(1000.0);
+  FACT(1e12d), CUBE(1e9d), DIMENSION(1000.0), SYSTEM(1000.0);
```

Comment:
Are these HEX costs ? Were these chosen based on some planner analysis that we can learn from ?

Reply:
I didn't want to write 1 followed by 12 zeros., thats just the double scientific notation (https://docs.oracle.com/javase/7/docs/api/java/text/DecimalFormat.html). 
In order for the union query to be chosen I had have the cube table have relatively less number of rows than the fact table. I also had to increase the number of rows of the fact and cube table to get the KuduProjectRel to be chosen.